linux/keyman-config/keyman_config/downloadkeyboard.py

```diff
@@ -28,21 +28,13 @@ def __init__(self, view=None):
         vbox = Gtk.Box(orientation=Gtk.Orientation.VERTICAL)
 
         s = Gtk.ScrolledWindow()
-        # TODO update (or remove) user_agent once website supports Linux kmp packages
-        user_agent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36"
-        #user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36"
         webview = WebKit2.WebView()
-        settings = WebKit2.Settings()
-        settings.set_user_agent(user_agent)
-        webview.set_settings(settings)
         webview.connect("decide-policy", self.keyman_policy)
-        # TODO update website URI once website supports Linux kmp packages
-        webview.load_uri("https://keyman.com/keyboards?embed=macos&version=10")
+        webview.load_uri("https://keyman.com/keyboards?embed=linux&version=10")
```

Comment:
Does this version need to update with each stable release?

Any way to have the CI populate it, or default to /resources/VERSION.md ?

Reply:
good idea. It now defaults to the major version in resources/VERSION.md and is set during make reconf

Reply:
We use `https://keyman.com/go/` URLs exclusively in Keyman Desktop on Windows. These then redirect internally. I'd like to do the same for Keyman for Linux; that way we have a set of URLs that we can maintain independently even if we change the rest of the site around.

Reply:
For example, the Keyman Desktop 10 download dialog is at: `https://keyman.com/go/desktop/10.0/download-keyboards?version=<xsl:value-of select="/Keyman/Version" />` (where `10.0` actually happens to be the URL version...)

Reply:
Sounds like we need to create keyman.com/go/linux/11.0/web.config to have a rule
```
                <rule name="/go/linux/11.0/download-keyboards" stopProcessing="true">
                  <match url="^download-keyboards" />
                  <action type="Redirect" url="/keyboards?embed=linux" />
                </rule>

```

fwiw, `embed=windows` displays a long list of languages, while `embed=linux` just has a search box.

Reply:
@darcywong00 just the search box is deliberate and is the same as mac. The list in windows is unsearchable.


`https://keyman.com/go/` sounds good so I can change to that once it is active for Linux

Reply:
See sillsdev/keyman-keyman.com#224

Reply:
it now uses the keyman.com go link